cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -55,20 +51,12 @@ def move(src, dest):
 
 extra_context["placeholder_repo_name"] = python_placeholder_repo_name
 
-if import_template_from_github:
-    directory = "python-template"
-    cookiecutter(
-        'git@github.com:edx/edx-cookiecutters.git',
-        extra_context=extra_context,
-        no_input=True,
-        directory=directory
-        )
-else:
-    cookiecutter(
-        os.path.join(EDX_COOKIECUTTER_ROOTDIR, 'python-template'),
-        extra_context=extra_context,
-        no_input=True
-        )
+cookiecutter(
+    EDX_COOKIECUTTER_ROOTDIR,
```

Comment:
With the two changes above, plus the cleanup of the `EDX_COOKIECUTTER_ROOTDIR` var, we can just make a single call to `cookiecutter`, without worrying if it's a local or remote call :)